script/deployPoolWithRenzoStabilityAndMintLiquidity.s.sol

```diff
@@ -0,0 +1,241 @@
+
+contract CreatePoolAndAddLiquidityScript is Script, Constants, Config {
+    using CurrencyLibrary for Currency;
+
+    /////////////////////////////////////
+    // --- Parameters to Configure --- //
+    /////////////////////////////////////
+
+    // TODO: configure 0 values
+
+    // Hook configuration
+    IRateProvider rateProvider =
+        IRateProvider(0xDb6df3559D2d96985062F0824442550CA7715960);
+    uint24 minFee = 100;
+    uint24 maxFee = 10_000;
+    address ezETH = 0x2416092f143378750bb29b79eD961ab195CcEea5;
+    address payable recipient =
+        payable(0xAdef586efB3287Da4d7d1cbe15F12E0Be69e0DF0);
+
+    // --- pool configuration --- //
+    // fees paid by swappers that accrue to liquidity providers
+    uint24 lpFee = LPFeeLibrary.DYNAMIC_FEE_FLAG; // Dynamic fee through hook
+    int24 tickSpacing = 1;
+
+    // starting price of the pool, in sqrtPriceX96
+    uint160 startingPrice;
+
+    // --- liquidity position configuration --- //
+    uint256 public token0Amount = 1e18;
+    uint256 public token1Amount;
+
+    // range of the position
+    int24 tickLower = -10; // must be a multiple of tickSpacing
+    int24 tickUpper = 10;
+    /////////////////////////////////////
+
+    function run() external {
+        startingPrice = SqrtPriceLibrary.exchangeRateToSqrtPriceX96(
+            rateProvider.getRate()
+        );
+
+        token1Amount = (token0Amount * 1e18) / rateProvider.getRate();
+
+        // deployHook;
+        hookContract = IHooks(_deployHook());
+
+        // tokens should be sorted
+        PoolKey memory pool = PoolKey({
+            currency0: currency0,
+            currency1: currency1,
+            fee: lpFee,
+            tickSpacing: tickSpacing,
+            hooks: hookContract
+        });
+        bytes memory hookData = new bytes(0);
+
+        // --------------------------------- //
+
+        // Converts token amounts to liquidity units
+        uint128 liquidity = LiquidityAmounts.getLiquidityForAmounts(
+            startingPrice,
+            TickMath.getSqrtPriceAtTick(tickLower),
+            TickMath.getSqrtPriceAtTick(tickUpper),
+            token0Amount,
+            token1Amount
+        );
+
+        // slippage limits
+        uint256 amount0Max = token0Amount + 1 wei;
+        uint256 amount1Max = token1Amount + 1 wei;
+
+        (
+            bytes memory actions,
+            bytes[] memory mintParams
+        ) = _mintLiquidityParams(
+                pool,
+                tickLower,
+                tickUpper,
+                liquidity,
+                amount0Max,
+                amount1Max,
+                recipient,
+                hookData
+            );
+
+        // multicall parameters
+        bytes[] memory params = new bytes[](2);
+
+        // initialize pool
+        params[0] = abi.encodeWithSelector(
+            posm.initializePool.selector,
+            pool,
+            startingPrice,
+            hookData
+        );
```

Comment:
harmless since `hookData` is zero bytes, but `initializePool`'s function signature takes a poolkey and uint160 sqrtPrice only

it was a legacy feature that we forgot to cleanup

Reply:
removed `hookData` param from initializePool  arguments.